libraries/AC_AttitudeControl/AC_AttitudeControl_Multi_6DoF.h

```diff
@@ -20,7 +20,7 @@ class AC_AttitudeControl_Multi_6DoF : public AC_AttitudeControl_Multi {
 
     // Command a Quaternion attitude with feedforward and smoothing
     // not used anywhere in current code, panic so this implementaiton is not overlooked
-    void input_quaternion(Quaternion attitude_desired_quat) override;
+    void input_quaternion(Quaternion& attitude_desired_quat, Vector3f ang_vel_target) override;
```

Comment:
```suggestion
    void input_quaternion(const Quaternion& attitude_desired_quat, const Vector3f &ang_vel_target) override;
```

Reply:
I can do this for ang_vel_target but attitude_desired_quat isn't constant.

Reply:
opps, no I can't. It is changed when limited.